lib/src/main/java/com/diffplug/spotless/npm/PlatformInfo.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2016 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.spotless.npm;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Locale;
+
+class PlatformInfo {
+	private PlatformInfo() {
+		// no instance
+	}
+
+	static OS normalizedOS() {
```

Comment:
The result could be cached.

Reply:
Yeah it could, but I'd consider this premature optimization: the method is called 1-2 times per spotless run...

Reply:
It was not about optimisation, it's just "pointless" to determine it again, it will not change.

Sure, I'm OK with that.